docs/DafnyRef/Attributes.md

```diff
@@ -208,20 +203,7 @@ What this is saying is that the quantified expression, `f(u) == u + r`,
 which may depend on the heap, is also valid for any good heap that is either the
 same as the current heap, or that is derived from it by heap update operations.
 
-### 22.1.9. imported
```

Comment:
Does this still exist?

Reply:
Nope, I checked with @RustanLeino  and it does not exists anymore. People can write their own attributes if they want, but "imported" is not an attribute supported by Dafny